### PR TITLE
Remove obsolete code for the `auth`, `cluster` and `config` command.

### PR DESCRIPTION
**Pull request by rgo3**
This commit removes obsolete code which is replaced by the new
`dcos-cli` written in Go. Respective integration tests have been removed
in cc967b5ac427cc11f96d2a2261e2ecc9ad18f95c already.

https://jira.mesosphere.com/browse/DCOS_OSS-4288

### CODE REVIEW

**Comment by rgo3** [comment]
@mesosphere-ci run all integration tests

**Comment by rgo3** [comment]
@mesosphere-ci run all integration tests

**Review by leehambley — APPROVED** [review]
LGTM

**Comment by leehambley** [comment]
@mesosphere-ci run all integration tests

**Comment by armandgrillet** [comment]
@mesosphere-ci run windows integration tests

**Comment by rgo3** [comment]
@mesosphere-ci run windows integration tests

**Comment by rgo3** [comment]
@mesosphere-ci run windows integration tests

**Comment by rgo3** [comment]
@mesosphere-ci run windows integration tests

**Comment by rgo3** [comment]
@mesosphere-ci run windows integration tests

**Comment by rgo3** [comment]
@mesosphere-ci run windows integration tests

**Comment by rgo3** [comment]
@mesosphere-ci run windows integration tests

**Comment by rgo3** [comment]
@mesosphere-ci run windows integration tests

**Comment by rgo3** [comment]
@mesosphere-ci run windows integration tests

**Comment by rgo3** [comment]
@mesosphere-ci run windows integration tests

**Comment by rgo3** [comment]
@mesosphere-ci run windows integration tests

**Comment by rgo3** [comment]
@mesosphere-ci run all integration tests

File: ci/integration-tests-windows.groovy [hunk 1]
```diff
@@ -72,13 +72,14 @@ node('py36') {
...
                                 cd python/lib/dcoscli; \
-                                python -c 'import urllib.request; urllib.request.urlretrieve(\\\"https://downloads.dcos.io/binaries/cli/windows/x86-64/latest/dcos.exe\\\", \\\"dist/dcos.exe\\\")'; \
-                                dist/dcos cluster remove --all; \
-                                dist/dcos cluster setup ${DCOS_TEST_URL} --insecure; \
-                                dist/dcos plugin add ../../../build/windows/dcos-core-cli.zip; \
-                                PATH=$PWD/dist:$PATH; \
                                 make env; \
-                                ./env/Scripts/tox -e py35-integration"
+                                rm -f ./env/Scripts/dcos.exe; \
+                                python -c 'import urllib.request; urllib.request.urlretrieve(\\\"https://downloads.dcos.io/cli/testing/binaries/dcos/windows/x86-64/master/dcos.exe\\\", \\\"dist/dcos.exe\\\")'; \
```
Comment (bamarni) [inline ci/integration-tests-windows.groovy:77]: not directly related to this PR, but we should also update this URL on Linux and mac

Comment (rgo3) [reply ci/integration-tests-windows.groovy:77]: I can do a PR for that now.

**Review by armandgrillet — APPROVED** [review]

**Pull request merged**